templates/workspace_services/guacamole/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.3.0"
+__version__ = "0.3.1"
```

Comment:
Shouldn't it be the same as in : templates/workspace_services/guacamole/porter.yaml? Mmmm I wonder how did it work until now, or maybe I  am missing something....